pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1507,6 +1507,12 @@ self: super: {
   # Fixed at head, but hasn't cut a release in awhile.
   darcs = doJailbreak super.darcs;
 
+  # Test suits dependencies are outdated
+  typerep-map = dontCheck super.typerep-map;
+
+  # Test suites are not implemented
```

Comment:
What does this comment mean?

Reply:
typerep-map's test suite failed because dependencies could not be satisfied (QuichCheck etc. are out-dated)
co-log-sys's test suite failed because it uses IO while also enables NoImplicitPrelude.